docs/vector_store.ipynb

```diff
@@ -563,6 +560,115 @@
     "\n",
     "print(docs)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a Vector Store using existing table\n",
+    "\n",
+    "A Vector Store can be built up on an existing table.\n",
+    "\n",
+    "Assuming there's an already existing table in Alloy DB: `products`, which stores product details for an eComm venture.\n",
+    "\n",
+    "The table has been created using following SQL:\n",
+    "\n",
+    "```\n",
+    "-- Sample create table SQL for table creation\n",
+    "CREATE TABLE products (\n",
+    "    product_id SERIAL PRIMARY KEY,\n",
+    "    name VARCHAR(255) NOT NULL,\n",
+    "    description TEXT,\n",
+    "    price_usd DECIMAL(10, 2) NOT NULL,\n",
+    "    category VARCHAR(255),\n",
+    "    quantity INT DEFAULT 0,\n",
+    "    sku VARCHAR(255) UNIQUE NOT NULL,\n",
+    "    embed vector(768) DEFAULT NULL, --> vector dimensions depends on the embedding model\n",
+    "    image_url VARCHAR(255)\n",
+    ");\n",
+    "\n",
+    "-- Insertion of records\n",
```

Comment:
Let's not highlight the use case of inserting NULL embeddings. See comment on fully working example